pkgs/development/python-modules/signedjson/default.nix

```
@@ -1,24 +1,24 @@
 { lib
 , buildPythonPackage
-, fetchFromGitHub
+, fetchPypi
 , canonicaljson
 , unpaddedbase64
 , pynacl
 , typing-extensions
+, setuptools-scm
+, importlib-metadata
 }:
 
 buildPythonPackage rec {
   pname = "signedjson";
   version = "1.1.1";
 
-  src = fetchFromGitHub {
-    owner = "matrix-org";
-    repo = "python-${pname}";
-    rev = "v${version}";
-    sha256 = "0y5c9v4vx9hqpnca892gc9b4xgs4gp5xk6l1wma5ciz8zswp9yfs";
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0280f8zyycsmd7iy65bs438flm7m8ffs1kcxfbvhi8hbazkqc19m";
   };
 
-  propagatedBuildInputs = [ canonicaljson unpaddedbase64 pynacl typing-extensions ];
+  propagatedBuildInputs = [ canonicaljson unpaddedbase64 pynacl typing-extensions setuptools-scm importlib-metadata ];
```

```suggestion
  nativeBuildInputs = [ setuptools-scm ];
  propagatedBuildInputs = [ canonicaljson unpaddedbase64 pynacl typing-extensions  importlib-metadata ];
```
shouldn't be needed at runtime